### PR TITLE
Updated Entity team split

### PR DESCRIPTION
**Pull request by Rajandeep98**
updated entity team split to their respective matrics

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-57-6mwlug86.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-57-49tk951z.web.app

**Review by bolyachevets — COMMENTED** [review]
The preview URL for release-notes generates an error:

Screenshot 2024-02-22 at 11 01 47 AM

Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'nodes')
    at Pu (index.a4bb979e.js:113:344)
    at async Proxy.created (index.a4bb979e.js:113:1230)

File: apps/release-notes/composable/classifyRelease.ts [hunk 1]
```diff
@@ -21,8 +21,8 @@ export async function classifyReleases (
     endCursor: ''
   }
   if (team === 'ENTITIES') {
-    myGhIds = await getBoard('Entities Team Space - new')
-  } else if (team === 'NAMETEAMSPACE') {
+    myGhIds = await getBoard('entities---david')
```
Comment (bolyachevets) [inline apps/release-notes/composable/classifyRelease.ts]: these are the entity board names in zenhub:

Screenshot 2024-02-22 at 11 19 01 AM

**Review by bolyachevets — CHANGES_REQUESTED** [review]

File: apps/release-notes/composable/classifyRelease.ts (hunk 1 above)
Comment (Rajandeep98) [reply apps/release-notes/composable/classifyRelease.ts]: they got divided into 2 teams

Comment (Rajandeep98) [reply apps/release-notes/composable/classifyRelease.ts]: oh i see i lost some changes while pushing

Comment (bolyachevets) [reply apps/release-notes/composable/classifyRelease.ts]: yes. and the team names are 'Entities - David' and 'Entities - Olga', not 'entitities---david'

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-57-49tk951z.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-57-6mwlug86.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-57-49tk951z.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-57-6mwlug86.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-57-49tk951z.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-57-6mwlug86.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://application-status-dev--pr-57-6mwlug86.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://release-notes-dev--pr-57-49tk951z.web.app

**Review by bolyachevets — APPROVED** [review]

**Pull request merged**